src/polyrith.lean

```diff
@@ -4,6 +4,24 @@ import system.io
 import algebra
 import tactic.linear_combination
 
+/--
```

Comment:
These are functions that aren't unique to `polyrith`, I've already made a PR to mathlib adding them.